care/facility/api/serializers/patient_consultation.py

```diff
@@ -57,6 +61,18 @@
 MIN_ENCOUNTER_DATE = make_aware(settings.MIN_ENCOUNTER_DATE)
 
 
+class ConsultationSymptomsSerializers(serializers.ModelSerializer):
+    class Meta:
+        model = ConsultationSymptoms
+        fields = ("symptom", "other_symptom", "onset_date", "cure_date", "consultation")
```

Comment:
see how other serializers are done. Let's serialize all fields, including the id (external_id), created_by, modified_by, modified and created date, except `deleted`.

Reply:
@rithviknishad fine but first the symptoms should get created then we can think of it. I have made migrations and migrated as well but still maybe while passing the attributes in the ConsultationSymptoms() object in the bulk create its unable to identify symptom field from the model